utils/uiTypes/VotePlugin.ts

```diff
@@ -248,25 +261,35 @@ export class VotingClient {
         clientProgramId,
         instructions
       )
-      const remainingAccounts: AccountData[] = []
-      for (let i = 0; i < this.votingNfts.length; i++) {
-        const nft = this.votingNfts[i]
-        const tokenAccount = await nft.getAssociatedTokenAccount()
 
-        remainingAccounts.push(
-          new AccountData(tokenAccount),
-          new AccountData(nft.address)
+      if (!ON_NFT_VOTER_V2) {
+        console.log('on nft voter v1')
+        const updateVoterWeightRecordIx = await getUpdateVoterWeightRecordInstruction(
+          this.client,
+          walletPk,
+          registrar,
+          voterWeightPk,
+          this.votingNfts,
+          type
         )
+        instructions.push(updateVoterWeightRecordIx)
+      } else {
+        console.log('on nft voter v2')
+        const {
+          createNftTicketIxs,
+          updateVoterWeightRecordIx,
+        } = await getUpdateVoterWeightRecordInstructionV2(
+          this.client,
+          walletPk,
+          registrar,
+          voterWeightPk,
+          this.votingNfts,
+          type
+        )
+        createNftActionTicketIxs?.push(...createNftTicketIxs)
```

Comment:
why is `createNftActionTicketIxs` needed instead of just adding it to `instructions`?

Reply:
i guess it just makes it easier to batch the txs how you want? whatever, it doesnt really matter, this file is way past the point of no return on code quality anyway